server/app/controllers/admin/AdminQuestionController.java

```diff
@@ -369,17 +369,18 @@ private void updateDefaultLocalizations(
     }
 
     if (questionForm instanceof MultiOptionQuestionForm) {
-      final MultiOptionQuestionDefinition definition;
+      final ImmutableList<QuestionOption> updatedQuestionOptions;
       try {
-        definition = (MultiOptionQuestionDefinition) questionForm.getBuilder().build();
+        updatedQuestionOptions =
+            ((MultiOptionQuestionDefinition) updatedQuestionDefinitionBuilder.build()).getOptions();
```

Comment:
Instead of building the QuestionDefinition again, which increments nextAvailableId again, just use the QuestionDefinition we already have to get the new options.